docs/src/content/docs/ingest/ingest-genomic.mdx

```diff
@@ -1,66 +1,78 @@
 
 :::caution
-Genomic data cannot be ingested until the clinical data that the genomic files link to is complete. Please ensure clinical ingest is complete before attempting genomic ingest.
+Genomic data cannot be ingested until the clinical data that the genomic files link to is complete. Please ensure clinical ingest is complete before attempting ingest.
 :::
 
-An example post request would look something like the below which would ingest the pointers to a gzipped VCF file along with its associated tabix index file which are located within s3 storage and link it with the clinical Sample Registration sample_registration_id_1 :
+The format of the ingest request is specified by the SequencingIngest schema in candigv2-ingest. The schema specifies two types of objects:
+  * Experiments link the sequencing centre's names with the MoHCCN clinical data's Sample Registration identifiers
+  * Analyses describe the particular downstream analysis performed, along with the files associated with the analysis.
 
-```bash
-curl -X 'POST' \
- $CANDIG_URL'/ingest/genomic' \
- -H 'accept: application/json' \
- -H 'Content-Type: application/json' \
- -H 'Authorization: Bearer '$TOKEN
- -d '[ \
- { \
-    "program_id": "string", \
-    "analysis_id": "HG00096", \
-    "metadata": { \
-      "sequence_type": "wgs", \
-      "data_type": "variant", \
-      "reference": "hg38" \
-    }, \
-    "main": { \
-      "name": "string", \
-      "access_method": "s3://s3.us-east-1.amazonaws.com/1000genomes/HG00096.vcf.gz" \
-    }, \
-    "index": { \
-      "name": "string", \
-      "access_method": "s3://s3.us-east-1.amazonaws.com/1000genomes/HG00096.tbi" \
-    }, \
-    "samples": [ \
-    { \
-      "submitter_sample_id": "sample_registration_id_1", \
-      "analysis_sample_id": "HG00096" \
-    } \
-    ]}]' \
+For example, a request which would ingest a sequence variation analysis would look like below. It links the clinical Sample Registration sample_registration_id_1 with a sequencing centre Experiment and associates the analysis with the pointers to a gzipped VCF file along with its associated tabix index file which are located within s3 storage:
+
+```json
+{
+  "experiments": [
+    {
+      "program_id": "string",
+      "experiment_id": "SEQ_0001",
+      "submitter_sample_id": "LOCAL-sample_registration_id_1",
+      "metadata": {
+        "library_strategy": "WGS"
+      }
+    }
+  ],
+  "analyses": [
+    {
+      "program_id": "string",
+      "analysis_id": "HG00096",
+      "metadata": {
+        "analysis_type": "sequence_variation",
+        "reference": "hg38"
+      },
+      "main": {
+        "name": "string",
+        "access_method": "s3://s3.us-east-1.amazonaws.com/1000genomes/HG00096.vcf.gz"
+      },
+      "index": {
+        "name": "string",
+        "access_method": "s3://s3.us-east-1.amazonaws.com/1000genomes/HG00096.tbi"
+      },
+      "samples": [
```

Comment:
I am still stuck on whether this should be called `samples` or `experiments`

Reply:
we need more opinions